test/e2e/test/default.go

```diff
@@ -17,7 +18,12 @@ func DefaultSecurityContext() *corev1.PodSecurityContext {
 	}
 
 	if !Ctx().OcpCluster {
-		defaultUserID := int64(1000)
+		defaultUserID := int64(123456) // arbitrary user ID
+		// before 7.9.0 Stack images expected to run with the user ID 1000 or 0 (https://github.com/elastic/beats/issues/18871)
```

Comment:
Regarding EnterpriseSearch, I think https://github.com/elastic/ent-search/pull/2439 is only available starting 7.11